src/main/kotlin/no/nav/klage/oppgave/api/KlagebehandlingController.kt

```diff
@@ -31,25 +30,30 @@ class KlagebehandlingController(
 
     @GetMapping("/klagebehandlinger/{id}")
     fun getKlagebehandling(
-        @PathVariable("id") oppgaveId: Long,
-        request: HttpServletRequest
+        @PathVariable("id") klagebehandlingId: String
     ): KlagebehandlingView {
-        val innloggetIdent = innloggetSaksbehandlerRepository.getInnloggetIdent()
         logger.debug(
             "getKlagebehandling is requested by ident {} for oppgaveId {}",
-            innloggetIdent,
-            oppgaveId
+            innloggetSaksbehandlerRepository.getInnloggetIdent(),
+            klagebehandlingId
         )
-        return klagebehandlingFacade.getKlagebehandling(oppgaveId).also {
+        return klagebehandlingFacade.getKlagebehandling(klagebehandlingId.toUUIDOrException()).also {
             auditLogger.log(
```

Comment:
Auditlogging samtidig ja. Niiiice :)

Reply:
Hvorfor denne endringen (på static import)? Noe automatisk fra Intellij, eller noe foretrukket for lesbarhet?

Reply:
Hvilken endring på static import? Jeg måtte gjøre en manuell merge, men prøvde å ta koden din så rett inn som mulig.

Reply:
Å, nå så jeg det! Det er nok Intellij som har en tendens til å gjøre sånt når man kopierer kode. Jeg kan fikse det!